src/Management/test/Endpoint.Test/Actuators/Health/HealthEndpointOptionsTest.cs

```diff
@@ -17,7 +17,8 @@ public void Constructor_InitializesWithDefaults()
 
         Assert.Null(options.Enabled);
         Assert.Equal("health", options.Id);
-        Assert.Equal(ShowDetails.Never, options.ShowDetails);
+        Assert.Equal(ShowValues.Never, options.ShowComponents);
+        Assert.Equal(ShowValues.Never, options.ShowDetails);
```

Comment:
Not all members are asserted on.

From base type:
- Enabled
- Id
- Path
- RequiredPermissions
- AllowedVerbs
- RequiresExactMatch()

From this type:
- ShowComponents
- ShowDetails
- Claim
- Role
- Groups